src/inmanta/env.py

```diff
@@ -719,6 +701,24 @@ def init_env(self) -> None:
                         "is different from the Python version of this process."
                     )
 
+            raise VenvCreationFailedError(
+                msg=f"Unable to create new virtualenv at {self.env_path} (environment already exists)"
+            )
```

Comment:
I don't think this is right. If the environment already exists we should just continue.

Reply:
I just noticed this mistake